### PR TITLE
Prevent frontend from crashing on empty signalR message

### PR DESCRIPTION
**Pull request by Eddasol**
Solves #1537

### CODE REVIEW

File: frontend/src/components/Contexts/SignalRContext.tsx [hunk 1]
```diff
@@ -81,7 +81,14 @@ export const SignalRProvider: FC<Props> = ({ children }) => {
     }, [accessToken])
 
     const registerEvent = (eventName: string, onMessageReceived: (username: string, message: string) => void) => {
-        if (connection) connection.on(eventName, (username, message) => onMessageReceived(username, message))
+        if (connection)
+            connection.on(eventName, (username, message) => {
+                if (message === 'null') {
```
Comment (andchiind) [inline frontend/src/components/Contexts/SignalRContext.tsx:86]: Is the message a string with just "null" in it, and not a null string?

**Comment by andchiind** [comment]
This PR solves the error checking in the frontend, which is good. The underlying backend issue is likely this: https://github.com/equinor/flotilla/issues/1604

File: frontend/src/components/Contexts/SignalRContext.tsx (hunk 1 above)
Comment (Eddasol) [reply frontend/src/components/Contexts/SignalRContext.tsx:86]: Yes, the message is 'null' and its only after JSON.parse that we end up with null

**Review by oysand — APPROVED** [review]
LGTM

**Pull request merged**